play/manual/README.md

```diff
@@ -0,0 +1,32 @@
+# Manual Setup
+
+This directory contains a few scripts and corresponding config file templates to
+semi-manually setup and run a set of keypers locally. Basic instructions:
+
+0. Build `rolling-shutter` with `make -C ../../rolling-shutter build`.
+1. Run a postgres database, e.g. with `process-compose up` in `/play`
+2. Set the variable `private_key` in `ethereum.sh` to a private key with funds
+   on Chiado.
+3. Clone the repos
```

Comment:
Could you also add here that we need to extract submodules for these two repos? `git submodule update --init --recursive` thanks